test/cuda/layers.jl

```diff
@@ -259,3 +259,39 @@ end
     end
   end
 end
+
+@testset "Embedding" begin
+  vocab_size, embed_size = 5, 2
+  m = Flux.Embedding(vocab_size, embed_size)
+
+  x = [1, 3, 5]
+  y = m(x)
+  m_g = m |> gpu
+  x_g = x |> gpu
+  y_g = m_g(x_g)
+  @test collect(y_g) == y
+  
+  gs = gradient(() -> sum(m(x)), params(m))
+  gs_g = gradient(() -> sum(m_g(x_g)), params(m_g))
+  @test collect(gs_g[m_g.weight]) ≈ gs[m.weight]
```

Comment:
```suggestion
  @test Array(gs_g[m_g.weight]) ≈ gs[m.weight]
```

Should this be added to `gpu_gradtest` ? We don't have any activations here, but we can run it through `identity`.